llvm/include/llvm/IR/DebugInfoMetadata.h

```diff
@@ -2145,6 +2145,13 @@ class DILocation : public MDNode {
     return 0;
   }
 
+  const DILocation *getOrCloneWithoutAtom() const {
```

Comment:
I think this could just be "getWithoutAtom", it's already implied with DIMetadata that "get" means "find me an existing metadata or create a new one".